smalldata_tools/ana_funcs/droplet.py

```diff
@@ -10,38 +10,41 @@ class dropletFunc(DetObjectFunc):
     """ 
     Parameters
     ----------
-    threshold : float (default = 10.0)
-         Treshold in sigma or ADU, depending on the value of the useRms parameters
-    thresholdLow : float (default = 3.0)
+    threshold : float (default = 1)
```

Comment:
Given that useRms defaults to True, a threshold of 1 means that you you keep 1/3 of all pixels which is really not a great choice. This should be at least 3 or we should charge useRms to default to False to get the approach people in condensed matter use as they consider all pixels as created equal (as opposued to what people in particular physics usually do). I'd prefer to default threshold to 3.

Reply:
Good point! I think that people are more used to think in threshold in ADU (even more so now that it is in keV). If you don't see an issue changing useRMS to False by default might be more in line with what people expect as the default behavior.
Changing it to 3 would be a decent choice for both cases btw.

Reply:
I can see your point, but changing the default to something less appropriate just because that if what 'people' expect does not feel great to me. I'm not sure what we should do.